helm/values.schema.json

```diff
@@ -0,0 +1,1856 @@
+                      ]
+                    },
+                    "required": [
+                      "trust_roots"
+                    ]
+                  },
+                  "cert": {
+                    "$id": "#/properties/validators/items/anyOf/2/properties/cert",
+                    "title": "The cert schema",
+                    "description": "TLS certificate if using a registry requiring a self-signed certificate.",
+                    "type": "string",
+                    "pattern": "(?:-+BEGIN\\sCERTIFICATE[-]+)\n(?:(?:[A-Za-z0-9+\\/\\s])*={0,2})\n(?:-+END\\sCERTIFICATE[-]+)"
+                  },
+                  "host": {
+                    "$id": "#/properties/validators/items/anyOf/2/properties/host",
+                    "title": "The host schema",
+                    "description": "Not yet implemented.",
+                    "type": "string",
+                    "pattern": "(([-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6})|(localhost))(\\:\\d{2,5})?"
+                  },
+                  "auth": {
+                    "type": "object",
+                    "$id": "#/properties/validators/items/anyOf/2/properties/auth",
+                    "title": "The auth schema",
+                    "description": "Authentication credentials if using a private registry instance.",
+                    "oneOf": [
+                      {
+                        "$id": "#/properties/validators/items/anyOf/2/properties/auth/OneOf/0",
+                        "type": "object",
+                        "title": "The first OneOf schema",
+                        "properties": {
+                          "secret": {
+                            "$id": "#/properties/validators/items/anyOf/2/properties/auth/properties/secret",
+                            "type": "string",
+                            "pattern": "[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*",
+                            "title": "The auth secret schema",
+                            "description": "Name of a Kubernetes secret containing authentication credentials.",
+                            "examples": [
+                              "my-cosign-auth-creds"
+                            ]
+                          }
+                        },
+                        "required": [
+                          "secret_name"
+                        ]
+                      },
+                      {
+                        "$id": "#/properties/validators/items/anyOf/2/properties/auth/OneOf/1",
+                        "type": "object",
+                        "title": "The second OneOf schema",
+                        "properties": {
+                          "k8s_keychain": {
+                            "$id": "#/properties/validators/items/anyOf/2/properties/auth/properties/k8s_keychain",
+                            "type": "boolean",
+                            "title": "The auth k8s_keychain schema",
```

Comment:
there is technically a default of `False` if not set at all, but I am not sure how you decided to handle this